.github/workflows/publish.yml

```diff
@@ -0,0 +1,33 @@
+name: Site publish
+
+on:
+    check_suite:
```

Comment:
@Westbrook I set this up to run when the check_suite completes, the goal being that if a commit message includes `#publish` and the check suite has passed, it will try to publish the docs site.

Reply:
Nice: do we need to change something about the publish script? That string doesn't seem available to-date: https://github.com/adobe/spectrum-web-components/commit/3be62133721efba844cd7032566a2c49ed6d9875